src/zarr/registry.py

```diff
@@ -266,4 +328,50 @@ def get_ndbuffer_class(reload_config: bool = False) -> type[NDBuffer]:
     )
 
 
+# TODO: merge the get_vXdtype_class_ functions
+# these can be used instead of the various parse_X functions (hopefully)
+def get_v3dtype_class(dtype: str) -> type[ZarrDType]:
```

Comment:
See the examples for how these can be used. As in the comment, the idea is for these to replace the various `parse_X` functions in the codebase.